tasks/ubuntu/trusty.task/os_boot.erb

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Could these new templates go in a new `bionic.task` directory? Or is it necessary to update the trusty and xenial files here?

Reply:
Oh, I think I get it -- did these templates change for bionic (which is now default) and now the trusty task needs its own copies?